Automattic/wp-calypso
# Login Assistance: Show migrate CTA when login fails

Fixes #90950

## Proposed Changes

* Adds a new migration notice (CTA) that is displayed on the login form when a user enters an unknown/unregistered email.

![image](https://github.com/Automattic/wp-calypso/assets/1612178/dc6975b8-96aa-4f33-bee5-ace7f70dc096)


## Why are these changes being made?


* This is part of the login assistance initiative (pdxWSz-1vb-p2). The aim is to show a migration nudge once the user enters a non-existing email address.

## Testing Instructions



* In an incognito window, go to http://calypso.localhost:3000/log-in and enter a non-existing email.
* Verify that only a "User does not exist." error is displayed.
* Close and re-open the page.
* Now manually assign yourself to the "Treatment" group of the experiment following this guide: PCYsg-SwK-p2 (Manually assign with Abacus assignment bookmarklet)
* After assigning yourself to the "Treatment" group, enter a non-existing email again.
* Verify that you see the "User does not exist." error and the migration notice.
* Verify that the migration notice CTA points to https://wordpress.com/move/.

## Pre-merge Checklist
